src/nyc_trees/apps/quiz/urls.py

```diff
@@ -3,16 +3,10 @@
 from __future__ import unicode_literals
 from __future__ import division
 
-from django.conf.urls import patterns, url
-
-from apps.quiz.routes import quiz_list_page, quiz_page
+from django.conf.urls import patterns
```

Comment:
This stub existed before I started working on quiz stuff. I don't have a strong opinion about its existence.